bundles/af-core/src/test/resources/form/formcontainer/exporter-formcontainerv2.json

```diff
@@ -9,6 +9,15 @@
       "dorTemplateRef": "xyz"
     },
     "fd:path": "/content/forms/af/demo/jcr:content/formcontainerv2",
+    "fd:signerInfo":{
+      "jcr:primaryType":"nt:unstructured",
```

Comment:
Why does exporter have these jcr:primary type ? Exporter represents business model and jcr structure